src/store/reducer/currentAdminSlice.ts

```diff
@@ -21,11 +21,11 @@ export const currentAdminSlice = createSlice({
   reducers: {
     setCurrentAdmin: (state, action: PayloadAction<currentAdmin>) => {
       state.currentAdmin = action.payload;
-      console.log("setCurrentAdmin", state.currentAdmin);
+      localStorage.setItem("adminInfo", JSON.stringify(state.currentAdmin));
     },
     clearCurrentAdmin: (state) => {
       state.currentAdmin = null;
-      console.log("clearCurrentAdmin", state.currentAdmin);
+      localStorage.removeItem("adminInfo");
```

Comment:
when you use admin's data, you get it or remove it from local storage, so its reducer doesn't any work, just get it from local storage directly when calling, no need to call reducer